docs/_static/vdr/v1.yaml

```diff
@@ -0,0 +1,231 @@
+            application/json:
+              $ref: '#/components/schemas/DIDResolutionResult'
+        "400":
+          description: "given DID is incorrect."
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: "DID couldn't be found."
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: "An error occurred while processing the request."
+          content:
+            text/plain:
+              schema:
+                type: string
+    put:
+      summary: "Updates a Nuts DID Document"
+      operationId: "updateDID"
+      tags:
+        - DID
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DIDUpdateRequest'
+      responses:
+        "200":
+          description: "DID Document has been updated."
+          content:
+            application/json+did-document:
+              schema:
+                 $ref: '#/components/schemas/DIDDocument'
+        "400":
+          description: "DID couldn't be updated because the input conflicts."
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: "DID couldn't be found."
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: "An error occurred while processing the request."
+          content:
+            text/plain:
+              schema:
+                type: string
+
+components:
+  schemas:
+    DIDDocument:
+      type: object
+      description: The actual DID Document according to the w3c spec.
+      required:
+        - id
+      properties:
+        assertionMethod:
+          description: List of KIDs that may sign JWTs, JWSs and VCs
+          type: array
+          items:
+            type: string
+        authentication:
+          description: List of KIDs that may alter DID documents that they control
+          type: array
+          items:
+            type: string
+        context:
+          description: Array of URIs
+          type: array
+          items:
+            type: string
+        controller:
+          description: Array of DIDs that have control over the DID Document
+          type: array
+          items:
+            type: string
+        id:
+          description: DID according to Nuts specification
+          example: "did:nuts:1"
+          type: string
+        service:
+          description: list of supported services by the DID subject
+          type: array
+          items:
+            $ref: '#/components/schemas/Service'
+        verificationMethod:
+          description: list of keys
+          type: array
+          items:
+            $ref: '#/components/schemas/VerificationMethod'
+    DIDDocumentMetadata:
```

Comment:
is this spec meant as implementation of the w3c spec or the nuts method? If the first, the nuts-specific properties shoudn't be in there (if it's the spec of the nuts method, it's fine)

Reply:
It's based on Nuts specs